website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -53,9 +51,9 @@ example, the `--domain` would be `example.com`.
 So let's create a new project with the same domain (`example.com`):
 
 ```sh
-$ mkdir nginx-operator
-$ cd nginx-operator
-$ operator-sdk init --plugins=helm --domain=example.com
```

Comment:
+1 to this. I've mentioned this in the [style guide issue](https://github.com/operator-framework/operator-sdk/issues/3615)